libcodechecker/server/permissions.py

```diff
@@ -75,6 +75,10 @@ def __init__(self, name, default_enable=True,
             if SUPERUSER not in managed_by:
                 managed_by = [SUPERUSER] + managed_by
 
+            # Product admin can manage another project admins.
```

Comment:
Please add a FIXME to refactor this into a better approach later on (e.g. a `manages_self` constructor parameter), because at first glance this is really hacky. (Even the superuser approach is hacky but the superuser *should* be the only special case.)

Otherwise it will work for now.

Reply:
I fixed it in this pull request.